morpheus/_lib/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ include(cmake/python_modules/common.cmake)
 #----------morpheus._lib.file_types---------
 include(cmake/python_modules/file_types.cmake)
 
+#----------morpheus._lib.filter_source---------
+include(cmake/python_modules/filter_source.cmake)
+
```

Comment:
I dont love that we have different compilation units and python modules for single enums. Can we combine this into a single module. Something like enums, data, or possible untify it with common (I forget why its separate from common)

Reply:
Done, merged both of them into common, which makes this a breaking change